test/unit/messaging/create-message.test.js

```diff
@@ -1,9 +1,9 @@
 const { BODY } = require('../../mocks/messaging/body')
 const { TYPE } = require('../../../app/constants/type')
 const { SOURCE } = require('../../../app/constants/source')
+const { SESSION_ID } = require('../../mocks/messaging/session-id')
```

Comment:
There's still constants in the middle of the mocks, I'd say move the constants to the top of the file and separate them with a space